modules/arrow/src/geoarrow/convert-geoarrow-to-binary-geometry.ts

```diff
@@ -21,6 +23,8 @@ type BinaryGeometryContent = {
   nDim: number;
   geomOffset: Int32Array;
   geometryIndicies: Uint16Array;
+  triangles?: Uint32Array;
```

Comment:
Nit: Again, can we add a `/** indices into the flatCoordinateArray */` type comment before every field here? that would be super helpful.